apps/demo-app/src/app/examples/search-filter/search-filter/search-filter-example.component.ts

```diff
@@ -19,6 +20,8 @@ import { of } from 'rxjs';
   styleUrls: ['./search-filter-example.component.scss']
 })
 export class SearchFilterExampleComponent {
+  @ViewChild(EntrySearchFilterComponent) entrySearchFilterComponent: EntrySearchFilterComponent;
```

Comment:
Whenever possible use { static: true } on the @ViewChild query. In this case we should use it.